<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reading</class>
 <widget class="QWidget" name="reading">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="Text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>571</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="xuanze">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>500</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentText">
    <string>A篇</string>
   </property>
   <item>
    <property name="text">
     <string>A篇</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B篇</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C篇</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>D篇</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前是</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="syt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>上一题</string>
   </property>
  </widget>
  <widget class="QPushButton" name="xyt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>10</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>下一题</string>
   </property>
  </widget>
  <widget class="QLabel" name="dn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text1">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>160</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonC">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>240</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="timu">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>490</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonD">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>410</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonB">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>250</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>400</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>320</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonA">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>180</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLabel" name="tishi">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>退出练习</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
